src/app.ts

```diff
@@ -320,8 +325,8 @@ function registerPagoPARoutes(
 ): void {
   const bearerTokenAuth = passport.authenticate("bearer", { session: false });
 
-  const pagopaController: PagoPAController = container.resolve(
-    PAGOPA_CONTROLLER
+  const pagopaController: PagoPAController = new PagoPAController(
+    PROFILE_SERVICE
```

Comment:
`PROFILE_SERVICE` should be passed as parameter to `registerPagoPARoutes`, you don't want to have global variables

Reply:
fixed in acedf9a